.github/workflows/docker-auto-setup.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Prepare Docker build-args
         id: build_args
         run: |
-          echo "::set-output name=github_sha_short::$(echo $GITHUB_SHA | cut -c 1-8)"
+          echo "::set-output name=github_sha_short::$(echo $GITHUB_SHA | cut -c 1-7)"
```

Comment:
you can get rid of this whole section by using `${GITHUB_SHA:0:7}` everywhere you use the output, which is a little simpler

Reply:
well, maybe not quite that simple since it's not always shell... how about just this:
```
echo "::set-output name=github_sha_short::${GITHUB_SHA:0:7}"
```